src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.243.0.mdx

```diff
@@ -0,0 +1,40 @@
+---
+subject: Browser agent
+releaseDate: "2023-10-04"
+version: 1.243.0
+features: ["Do not report ajax nodes in session traces if in deny list","Remove deprecated API inlineHit"]
+bugs: ["Re-update assets proxy to accept host URL strings","Adjust Session Replay meta and timestamps","Tap session entity into storage api for changes across tabs","Set Session Replay first chunk flags more reliably"]
+security: []
+---
+
+## v1.243.0
+
+### Features
+
+#### Do not report ajax nodes in session traces if in deny list
+Prevent ajax nodes derived from calls found in the ajax deny list from populating Session Trace payloads
+
+#### Remove deprecated API inlineHit
+Removing the browser agent API inlineHit. This API was deprecated in 2014 and is no longer supported or used by newer versions of the browser agent.
+
+### Bug Fixes
+
+#### Re-update assets proxy to accept host URL strings
+Tweaks the feature added in v1.240.0 wherein the agent code became proxy-able. It worked correctly in expecting that the protocol (i.e. `https://`) is included in the configuration input, but we have since decided that it can be excluded in conformity with the proxy beacon field that only accepts a host string.
```

Comment:
```suggestion
Updates the proxy-ability feature added in v1.240.0 of the agent code. It worked correctly in expecting that the protocol (i.e. `https://`) is included in the configuration input, but we have since decided that it can be excluded in conformity with the proxy beacon field that only accepts a host string.
```